.travis/run.sh

```diff
@@ -85,6 +85,60 @@ for BOARD in $BOARDS; do
 				PROG=$PROG BOARD=$BOARD TARGET=$TARGET make load || true
```

Comment:
FYI - This script is actually also used outside travis for building the Xilinx package....